content/foundations/color.mdx

```diff
@@ -2,65 +2,191 @@
-
-## Color roles 
-Color roles are defined by the meaning they convey in the UI. Primer has 8 color roles with 4 variants each. 
-
-![Color role anatomy: color role prefix + variant suffix](https://user-images.githubusercontent.com/6951037/126636938-8674c80c-c9cb-425c-b6d5-d1ddcc742fcb.png)
-
-### Global roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `neutral-` | Use to highlight content without any added meaning. |
-| `accent-` | Use to draw attention to interactive elements. |
-| `success-` | Use to expresses the completion or positive outcome of a task. |
-| `attention-` | Use to warn of pending tasks or highlight active content.  |
-| `severe-` | Use when there are more than 3 levels of states, for example in heatmaps. |
-| `danger-` | Use to inform of error or another negative message. |
-
-### GitHub-specific roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `done-` | Completion color for productivity and code review workflows. |
-| `sponsors-` | Color suite for all Sponsors-related content. |
-
-### Variants
-
-| Suffix | Role |
-| -------- | -------- |
-| `-fg` | Text and icons. |
-| `-emphasis` | High contrast for background and borders. |
-| `-emphasis-plus` | Highest contrast for background and borders. Use only with the `neutral-` prefix. |
-| `-muted` | Medium contrast alpha value, recommended for borders. |
-|`-subtle` | Low contrast solid color, recommended for backgrounds. |
+## How to use color for product UI
+Primer delivers colors in the form of [variables](https://primer.style/primitives/colors) or design tokens. Color variables can be grouped into different groups based on their purpose:
+
+- **Functional:** To convey a meaning or a state. For example, from the functional perspective the color green is used to reinforce positive messaging. In a functional system green variables are named with the suffix `.success`
+- **Presentational:** To represent a color. For example, the color steps in the Primer scale are named by color and lightness. Names are such as `scale.blue.5`. These variables don't currently support color modes. There is no available system to use colors in this form yet.
+
+As the system grows, we are working to provide different APIs that fit different color needs. Check the [Primitives](https://github.com/primer/primitives/releases/tag/v7.2.0) repository to follow along as we release new variables and systems.
+
+### Functional system
+The functional system is based on the meaning, or purpose, that colors have in the interface. This system is structured into two groups of variables:
+
+- **Foundations:** Foregrounds, backgrounds and borders that make 95% of a product interface. Each variable is generated based on visual weight. 
+- **Color roles:** Foregrounds, backgrounds and borders to highlight affordance or meaning of elements in the UI.
+
+<img width="960" alt="color-roles-list" src="https://user-images.githubusercontent.com/6951037/141302835-05ea2e26-fc89-49ed-8dc5-2af6a6ca9df8.png" />
```

Comment:
Is this image only related to the "color roles" above? If so, I think we need an image for the "foundations" as it's not clear at a glance and it feels like that's missing.

Reply:
Thinking more about this. I think it should be:

```
#### Foundations

-image here-

Text.

#### Color roles

-image here that you already have-

Text.
```

So instead of a list it's like this ^